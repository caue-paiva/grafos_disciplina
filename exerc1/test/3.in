5
7
1 8
4
2 1 2 3
8 1 2
8 2 3
4
2 2 3 5
5 2
7
0

Neste caso, sao realizadas diversas operacoes no grafo. Comecando com 5 vertices iniciais, e exibido o numero de vertices, adicionado um novo vertice, adicionadas arestas entre vertices com pesos especificos, verificado se certas arestas existem, exibidos os vertices adjacentes a um vertice especifico e, por fim, e exibido o numero de vertices novamente antes da saida do programa.