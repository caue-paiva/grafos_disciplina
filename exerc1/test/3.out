Digite o numero de vertices inicial do grafo: 
OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: O grafo tem 5 vertices.

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: Digite o numero do vertice que deseja adicionar: 
OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: 
     1  2  3  4  5  8  
     ------------------
1: [ -1 -1 -1 -1 -1 -1 ]
2: [ -1 -1 -1 -1 -1 -1 ]
3: [ -1 -1 -1 -1 -1 -1 ]
4: [ -1 -1 -1 -1 -1 -1 ]
5: [ -1 -1 -1 -1 -1 -1 ]
8: [ -1 -1 -1 -1 -1 -1 ]
Numero de vertices: 6 , Numero de arestas bi-direcionais: 0 

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: Digite os numeros dos vertices que deseja adicionar uma aresta: Digite o peso da aresta: 
OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: Digite os numeros dos vertices para verificar se existe aresta entre eles: Existe aresta.

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: Digite os numeros dos vertices para verificar se existe aresta entre eles: Nao existe aresta.

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: 
     1  2  3  4  5  8  
     ------------------
1: [ -1  3 -1 -1 -1 -1 ]
2: [  3 -1 -1 -1 -1 -1 ]
3: [ -1 -1 -1 -1 -1 -1 ]
4: [ -1 -1 -1 -1 -1 -1 ]
5: [ -1 -1 -1 -1 -1 -1 ]
8: [ -1 -1 -1 -1 -1 -1 ]
Numero de vertices: 6 , Numero de arestas bi-direcionais: 1 

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: Digite os numeros dos vertices que deseja adicionar uma aresta: Digite o peso da aresta: 
OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: Digite o numero do vertice que deseja mostrar a lista de adjacencia: 1 3 

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: O grafo tem 6 vertices.

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: 